crates/solver/src/settlement/settlement_encoder.rs

```diff
@@ -102,27 +115,75 @@ impl SettlementEncoder {
         let sell_token_index = self
             .token_index(order.order_creation.sell_token)
             .expect("missing sell token with price");
+        let (trade, execution) = match is_liquidity_order {
```

Comment:
Rust question: is `match {true, false}` more idiomatic than an `if .. else` block?

Reply:
I wouldn't say so. There is no big difference so it can be personal preference.